batch/batch/server/server.py

```diff
@@ -615,12 +615,14 @@ def wrapped(request, *args, **kwargs):
         parameters['spec'], kube.client.V1PodSpec)
 
     batch_id = parameters.get('batch_id')
-    if batch_id:
-        batch = await Batch.from_db(batch_id, user)
-        if batch is None:
-            abort(404, f'invalid request: batch_id {batch_id} not found')
-        if not batch.is_open:
-            abort(400, f'invalid request: batch_id {batch_id} is closed')
+    if batch_id is None:
```

Comment:
I'd prefer to make batch_id required in the request schema above.  That's fed into cerberus: http://docs.python-cerberus.org/en/stable/index.html.  I think you can change it to: `'batch_id': {'required': True, 'type': 'integer'}`: http://docs.python-cerberus.org/en/stable/validation-rules.html#required